### ISSUE TITLE
[NBug] Culture is not supported. (Parameter 'culture')4096 (0x10...

### ISSUE THREAD

**Issue by MarekKnapek** [gitextensions/gitextensions#10549]
### Environment

- Git Extensions 4.0.0.15569
- Build cba315df5366fca01240d93b6e5a5077fd5e6630
- Git 2.38.1.windows.1
- Microsoft Windows NT 10.0.20348.0
- .NET 6.0.10
- DPI 96dpi (no scaling)

### Issue description

```
System.Globalization.CultureNotFoundException: Culture is not supported. (Parameter 'culture')
4096 (0x1000) is an invalid culture identifier.
   at System.Globalization.CultureInfo..ctor(Int32 culture, Boolean useUserOverride)
   at System.Globalization.CultureInfo..ctor(Int32 culture)
   at System.Windows.Forms.InputLanguage.get_Culture()
   at System.Windows.Forms.InputLanguageChangedEventArgs..ctor(InputLanguage inputLanguage, Byte charSet)
   at System.Windows.Forms.InputLanguage.CreateInputLanguageChangedEventArgs(Message m)
   at System.Windows.Forms.Control.WmInputLangChange(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.TextBoxBase.WndProc(Message& m)
   at System.Windows.Forms.TextBox.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)
```

Steps to reproduce:
 - GitExtensions v4.0.0.15569.
 - Windows Server 2022 x64 v10.0.20348.
 - Launch GitExtensions.
 - Change locale by pressing icon in Task Bar near clock.
 - Change locale back.
 - Dot Net exception is thrown.

Most likely not bug in GitExtensions, but in Windows or in DotNet.

### Steps to reproduce

.

**Comment by mstv** [comment]
Was it a locale different from English and Czech? Which?

**Comment by MarekKnapek** [comment]
My Windows interface and locale is in English. Not sure whether USA or Ireland or International. I changed it a lot in the past, because I wanted to use km/cm/kg instead of miles/inches/lbs. My keyboard layout is Czech (QWERTZ). When I invoked the Win+SpaceBar keyboard shortcut by mistake, some application started misbehaving (GitExtensions displayed error box and Visual Studio crashed). This is a new problem, it didn't cause any problems before, I guess some dotNet update caused it. Not sure GitExtensions can do anything about this.
